OpenEXR/IlmImf/ImfRgbaFile.cpp

```diff
@@ -167,9 +167,10 @@ cachePadding (ptrdiff_t size)
     // we are running on.  (It is ok if CACHE_LINE_SIZE is larger
     // than a real cache line.)
     //
+    // CACHE_LINE_SIZE = (1 << LOG2_CACHE_LINE_SIZE)
+    //
 
     static int LOG2_CACHE_LINE_SIZE = 8;
```

Comment:
Intel and AMD both have cache line size 64 (so log2 is 6, not 8).
Is there a reason that this is larger than the actual cache line size?